edit/edit.go

```diff
@@ -619,14 +659,25 @@ func edit(ed *Editor, cmd []rune) ([]rune, addr, error) {
 	case 'p':
 		return print(ed, a)
 	case '=':
-		if cmd[1] != '#' {
-			return nil, addr{}, errors.New("unknown command: " + string(c))
-		}
+		var ret string
 		at, err := where(ed, a)
 		if err != nil {
 			return nil, addr{}, err
 		}
-		return []rune(fmt.Sprintf("#%d,#%d", at.from, at.to)), at, nil
+		if len(cmd) == 1 || cmd[1] != '#' {
```

Comment:
Ehhh, that would really only make the edit function 5 lines shorter (the "if lineFrom == lineTo ...") I don't think it's worth it to move it to its own function, especially since we need to add the same check to the "=#" command.